### ISSUE TITLE
Cassandra cucmber tests fail (not starting in-memory db?)

### ISSUE THREAD

**Issue by mikehalma** [jhipster/generator-jhipster#2722]
On jhipster version 2.26.2 on Windoze, generate an app with:

```
? (3/15) Which *type* of authentication would you like to use? HTTP Session Authentication (stateful, default Spring Security mechanism)
? (4/15) Which *type* of database would you like to use? Cassandra
? (9/15) Do you want to use clustered HTTP sessions? No
? (10/15) Do you want to use WebSockets? No
? (11/15) Would you like to use Maven or Gradle for building the backend? Gradle
? (12/15) Would you like to use Grunt or Gulp.js for building the frontend? Grunt (recommended)
? (13/15) Would you like to use the LibSass stylesheet preprocessor for your CSS? No
? (14/15) Would you like to enable translation support with Angular Translate? No
? (15/15) Which testing frameworks would you like to use? Gatling, Cucumber
```

then run "gradle cucmberTest".

This fails and the stacktrace suggests that it is because the in-memory cassandra db is not started or not found?

```
Caused by: com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: localhost/127.0.0.1:9142 (com.datastax.driver.core.TransportException: [localhost/127.0.0.1:9142] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:227)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:82)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1307)
	at com.datastax.driver.core.Cluster.init(Cluster.java:159)
	at com.datastax.driver.core.Cluster.connect(Cluster.java:249)
	at com.datastax.driver.core.Cluster.connect(Cluster.java:282)
	at org.springframework.cassandra.config.CassandraCqlSessionFactoryBean.afterPropertiesSet(CassandraCqlSessionFactoryBean.java:82)
	at org.springframework.data.cassandra.config.CassandraSessionFactoryBean.afterPropertiesSet(CassandraSessionFactoryBean.java:43)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1637)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1574)
	... 80 more
```
Cheers,
Mike.

**Comment by jdubois** [comment]
I need to check the doc, but don't you need to have Cassandra running for doing those tests?

**Comment by pascalgrimaud** [comment]
No need to have Cassandra running to do tests
Cassandra Tests work fine.

**Cucumber** Cassandra Tests failed

**Comment by mikehalma** [comment]
I think I have fixed this - it wasn't starting the in memory Cassandra database. #2731

**Comment by pascalgrimaud** [comment]
We got a lot of issue related to HazelCast + Cucumber
Can you test HazelCast + Cucumber + Cassandra too plz ?

**Comment by mikehalma** [comment]
will do

**Comment by mikehalma** [comment]
OK my fix seems to be passing all tests.  Please see #2732

**Comment by pascalgrimaud** [comment]
closed by #2732
